app/components/nav-listing.js

```diff
@@ -0,0 +1,11 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  router: service(),
+
+  isUserRoute: computed('router.currentRouteName', function () {
+    return this.router.currentRouteName === 'users';
```

Comment:
Use `get` here `get(this, 'router.currentRouteName')`

Reply:
I have fixed this change.